teuthology/test/task/test_ceph_ansible.py

```diff
@@ -41,6 +41,15 @@ def fake_get_scratch_devices(remote):
         )
         self.patcher_get_scratch_devices.start()
 
+        def fake_teardown(self):
+            return
```

Comment:
I think it is better tested on real system than with mock, if you look at the teardown you will see it why. Also the way inheritance is done from the TestTask doesn't look clean to me.